docs/tutorials/scheduling.rst

```diff
@@ -0,0 +1,258 @@
+.. _scheduling_tutorial:
+
+.. doctest-skip-all
+
+***************************
+Scheduling an Observing Run
+***************************
+
+.. note::
+
+    Some terms used have been defined in :ref:`Terminology <terminology>`.
+
+Contents
+========
+
+* :ref:`scheduling-defining_targets`
+
+* :ref:`scheduling-creating_constraints_and_observing_blocks`
+
+* :ref:`scheduling-creating_a_transitioner`
+
+* :ref:`scheduling-scheduling`
+
+.. _scheduling-defining_targets:
+
+Defining Targets
+================
+
+We want to observe Deneb and M13 in the B, V and R filters. We are scheduled
+for the first half-night on July 6 2016 and want to know the order we should
+schedule the targets in.
+
+First we define our `~astroplan.Observer` object (where we are observing from):
+
+.. code-block:: python
+
+    >>> from astroplan import Observer
+
+    >>> apo = Observer.at_site('apo')
+
+Now we want to define our list of targets (`~astroplan.FixedTarget` objects),
+any object that is in SIMBAD can be called by an identifier.
+
+.. code-block:: python
+
+    >>> from astroplan import FixedTarget
+
+    >>> Deneb = FixedTarget.from_name('deneb')
+    >>> M13 = FixedTarget.from_name('m13')
```

Comment:
When you can, label variables with lowercase letters. Upper case first letters are preferred for classes.